src/freenas/usr/local/libexec/freenas-debug/network/network.sh

```diff
@@ -134,6 +134,10 @@ network_func()
 	iptables-save
 	section_footer
 
+	section_header "Iptables Rules with counters (iptables -L -v -n)"
```

Comment:
We call `iptables-save` right above this which is the same information just without the counters. Instead of calling this multiple times, you can probably just remove `iptables-save`